### ISSUE TITLE
Exponent operator not working?

### ISSUE THREAD

**Issue by Leeoku** [eslint/eslint#10213]
**Tell us about your environment**

* **ESLint Version:** 1.48
* **Node Version:** 8.11.1
* **npm Version:**5.6

**What parser (default, Babel-ESLint, etc.) are you using?** ESLINT 1.48

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
ESLint addon in VS Code
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
function square(num) {
  return num **2;
}
```

```bash
This was the eslint addon to VS Code
```

**What did you expect to happen?**
number should be squared, I believe this is new es6 syntax

**What actually happened? Please include the actual, raw output from ESLint.**
Error received: Parsing error: Unexpected token *

**Comment by platinumazure** [comment]
Hi @Leeoku, thanks for the issue.

The exponentiation operator is only available in ES2016 or later, so you would need to fix your ESLint configuration to use `ecmaVersion: 2016` (or later). I'm not sure how to do that in VS Code, though.

**Comment by not-an-aardvark** [comment]
Closing because the question has been answered.
